.travis.yml

```diff
@@ -18,8 +17,8 @@ install: ./gradlew -version
 
 #Don't build tags
 branches:
-  except:
-  - /^v\d/
+  only:
```

Comment:
as we will have githubactions for normal builds, we we only need this for the special case of "deployments" which should only happen on master